pkg/sloop/server/internal/config/config.go

```diff
@@ -78,6 +87,15 @@ func registerFlags(fs *flag.FlagSet, config *SloopConfig) {
 	fs.StringVar(&config.ApiServerHost, "apiserver-host", "", "Kubernetes API server endpoint")
 	fs.BoolVar(&config.WatchCrds, "watch-crds", true, "Watch for activity for CRDs")
 	fs.StringVar(&config.RestoreDatabaseFile, "restore-database-file", "", "Restore database from backup file into current context.")
+	fs.Float64Var(&config.BadgerDiscardRatio, "badger-discard-ratio", 0.01, "Badger value log GC uses this value to decide if it wants to compact a vlog file.  Smaller values free more disk space but use more computing resources")
```

Comment:
is this default too small? would 0.1 be sufficient?

Reply:
Sure, I can try out 0.1